pkg/kubelet/runtime.go

```diff
@@ -17,9 +17,11 @@ limitations under the License.
 package kubelet
 
 import (
-	"fmt"
 	"sync"
 	"time"
+
+	"github.com/pkg/errors"
```

Comment:
nit: Did you mean to use plain `"errors"` here? While the pkg/errors package is nice, I feel that we should be consistent with the surrounding code.